.dockerignore

```diff
@@ -5,17 +5,14 @@ cluster-provisioner
 kata-agent
 
 aws/
-azure/
 docs/
 install/
 ibmcloud/
 libvirt/
 peerpod-ctrl/
 peerpodconfig-ctrl/
-podvm/
 volumes/
 webwook/
 
-.git*
```

Comment:
I think `.git` is necessary to derive a VM image name from a commit hash string. 

https://github.com/confidential-containers/cloud-api-adaptor/blob/ade303e0acddcae9aa94bc753866578215587ae7/podvm/Makefile.inc#L37-L47

If `.dockerignore` excludes files that are not ignored by `.gitignore`, the result commit hash string will always contain `-dirty`.

This `.dockerignore` file is  first prepared for [`Dockerfile`](https://github.com/confidential-containers/cloud-api-adaptor/blob/ade303e0acddcae9aa94bc753866578215587ae7/Dockerfile) for the  `cloud-api-adaptor` container image. When building this image, commit hash is not calculated during container build, but is passed as argument to `docker build`.

https://github.com/confidential-containers/cloud-api-adaptor/blob/ade303e0acddcae9aa94bc753866578215587ae7/Makefile#L115-L116

https://github.com/confidential-containers/cloud-api-adaptor/blob/ade303e0acddcae9aa94bc753866578215587ae7/hack/build.sh#L33-L41

We can apply a similar approach for generating pod VM image names, but it needs more changes in the build scripts, so I think it is OK to make `.dockerignore` consistent with `.gitignore` for the time being.

We can implement such refactoring  in a later PR.

Reply:
>  so I think it is OK to make `.dockerignore` consistent with `.gitignore` for the time being.

Done